.github/workflows/Build.yml

```diff
@@ -97,3 +100,9 @@ jobs:
       - name: Publish the artifacts
         run: |
           gradle clean publish --stacktrace
+
+      - name: Tag Misk repo
+        uses: mathieudutour/github-tag-action@v6.1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          custom_tag: ${{ steps.assign_version.outputs.version }}
```

Comment:
Does this resolve to `GITHUB_OUTPUT` env variable?

Reply:
The `GITHUB_OUTPUT` env var is a magic one that exports things out of the step so it can be used by other steps. Docs at https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter